bower.json

```diff
@@ -27,14 +27,36 @@
     "package.json"
   ],
   "dependencies": {
-    "d2l-colors": "^2.3.0",
-    "d2l-offscreen": "^2.2.4",
-    "iron-iconset-svg": "polymerelements/iron-iconset-svg#^1.1.1",
-    "polymer": "^1.9.0"
+    "d2l-colors": "^3.0.1",
+    "d2l-offscreen": "^3.0.1",
```

Comment:
Should colors and offscreen be ranges as well? Just wondering how a `1.x` application would work, since it's going to be using the older version of colors. From [here](https://www.polymer-project.org/2.0/docs/devguide/hybrid-elements#dependency-variants):

>>> The default dependencies should use version ranges that include all versions supported by the component (typically, 1.9.1 or higher for Polymer itself).